cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/power/HostActivationExample.java

```diff
@@ -218,11 +218,19 @@ private Host createHost() {
 
         //Indicates if the Host will be powered on or not after creation
         final boolean activateHost = false;
+        
+        //Indicates Host Power On Delay
```

Comment:
For examples, is important to indicate the data unit.
This way, the comment could be something like that:

```java
//Indicates Host Power On Delay (in seconds)
```